code/game/atoms_movable.dm

```diff
@@ -335,6 +335,7 @@
 
 
 /atom/movable/proc/throw_at(atom/target, range, speed, thrower, spin)
+	set waitfor = FALSE
```

Comment:
I thought procs were `waitfor = FALSE` by default now?

Reply:
They are `set waitfor = TRUE` by default.
If you saw a line about this being deprecated, that's what it means.
It's worded in a confusing manner.